dashboard/pages/5_Word_Clouds.py

```diff
@@ -87,6 +83,26 @@ def clean_text(text: str, custom_stopwords: list) -> str:
     return lemmatized_words
 
 
+def filter_articles_by_date(articles: list, time_range: str):
+    """Filters the articles based on the selected time range."""
+
+    now = datetime.now()
+
+    if time_range == 0:
+        cutoff_date = now - timedelta(hours=1)
+    elif time_range == 1:
+        cutoff_date = now - timedelta(days=1)
+    elif time_range == 2:
+        cutoff_date = now - timedelta(days=7)
+    else:
+        return articles
+
+    filtered_articles = [
+        article for article in articles if article["date_published"] > cutoff_date.date()]
```

Comment:
not something to change but worth testing that edge cases (at the cutoff date) are treated as expected later on

Reply:
Good idea!